package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "portal-cli-app",
+  "version": "1.0.0",
+  "description": "Simple CLI for displaying and publishing dataset pages",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "chalk": "^4.1.0",
+    "commander": "^6.2.0",
+    "cpy": "^8.1.1",
+    "cross-spawn": "^7.0.3",
+    "figlet": "^1.5.0",
+    "listr": "^0.14.3",
+    "open": "^8.0.2",
+    "ora": "^5.1.0",
+    "prompts": "^2.4.0"
+  },
+  "files": [
+    "bin"
+  ],
+  "bin": {
+    "portal show": "index.js"
```

Comment:
I think `portal` command with no arguments should point to `index.js` where you can parse the arguments

Reply:
Okay, will update it now